start.py

```diff
@@ -54,12 +54,19 @@
 KUDU_PARCEL_REPO_URL = 'https://archive.cloudera.com/kudu/parcels'
 KUDU_PARCEL_VERSIONS = {'1.2.0': '5.10.0',
                         '1.3.0': '5.11.0',
-                        '1.4.0': '5.12.0'}
+                        '1.4.0': '5.12.0',
+                        '1.5.0': '5.13.0',
```

Comment:
Do we need to hardcode CDH-Kudu mappings for these versions where Kudu is part of CDH? I'd think we could find out the Kudu version included with CDH 5.13+ and then when we do validation, we'd just be able to stop people who try to use an invalid version. As it stands, we'd need to update this topology for every CDH release, which seems a bit painful.

Reply:
Done.